nil/tests/async_await/async_await_test.go

```diff
@@ -75,13 +75,15 @@ func (s *SuiteAsyncAwait) SetupSuite() {
 	}
 
 	const disableConsensus = true
-	s.Start(&nilservice.Config{
-		SplitShards:      false,
-		HttpUrl:          rpc.GetSockPath(s.T()),
-		NShards:          nShards,
-		ZeroState:        zeroState,
-		DisableConsensus: disableConsensus,
-	}, port)
+	s.Start(
+		&nilservice.Config{
+			SplitShards:      false,
+			HttpUrl:          rpc.GetSockPath(s.T()),
+			NShards:          nShards,
+			ZeroState:        zeroState,
+			DisableConsensus: disableConsensus,
+		},
+		port)
```

Comment:
Why did you change it?

Reply:
These are edits left over from adding another mandatory parameter with `Start`, which was later removed. Reverted the changes.